connector/src/main/java/com/google/cloud/spark/bigquery/v2/ParquetIntermediateRecordWriter.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.spark.bigquery.v2;
+
+import org.apache.avro.Schema;
+import org.apache.avro.generic.GenericRecord;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.Path;
+import org.apache.parquet.avro.AvroParquetWriter;
+import org.apache.parquet.hadoop.ParquetWriter;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+
+class ParquetIntermediateRecordWriter implements IntermediateRecordWriter {
```

Comment:
Do we have some benchmarks on how Parquet vs Avro for intermediate data? My guess is that writing Parquet is going to be much more expensive and was wondering if it made sense, given that it's a short-lived intermediate format.

Reply:
Based on the DSv1 data, parquet was faster for data sizes up to 250 GB or so, from there Avro was faster. I'll run a comparison test

Reply:
After running a benchmark, it appears that the Parquet writer as constantly slower than the avro one, so I've removed it.